tests/integration/test_notes.py

```diff
@@ -28,3 +29,19 @@ def test_it_returns_404_with_invalid_ticket(self, valid_header):
             headers=valid_header,
         )
         assert is_valid(response, 404)  # Bad Request- 'Invalid Ticket'
+
+    def test_it_deletes_one_note(
+        self, valid_header, create_note, create_admin_user, create_ticket
+    ):
+        note = create_note(create_admin_user(), create_ticket())
+
+        response = self.client.delete(
+            f"{self.endpoint}/{note.ticketid}/notes/{note.id}", headers=valid_header
+        )
+
+        assert response.json == {"message": "Note deleted"}
+
+        with pytest.raises(Exception) as e_info:
+            NotesModel.find(note.id)
+
+        assert "404 Not Found: Notes not found" in str(e_info.value)
```

Comment:
This is testing behavior in the BaseModel which is already thoroughly tested. Let's remove this test.